src/main/java/net/mcreator/generator/Generator.java

```diff
@@ -325,7 +325,7 @@ public void removeElementFilesAndLangKeys(GeneratableElement generatableElement)
 			List<GeneratorTemplate> globalTemplatesList = getGlobalTemplatesListForModElementType(type, performFSTasks,
 					templateID);
 
-			List<GeneratableElement> elementsList = workspace.getWorkspaceInfo().getElementsOfType(type).stream()
+			List<GeneratableElement> elementsList = workspace.getModElements().stream().filter(e -> e.getType() == type)

```

Comment:
While I agree WorkspaceInfo should only be used in FTL, this change does not belong to this PR and should have its own PR.